### DISCUSSION TITLE
Login Page first

### DISCUSSION THREAD

**Discussion by Stoufiler** [mealie-recipes/mealie#1720] [Q&A] [answered]
Hi there !

I allow myself to come to you to find out if it was possible to set up a login page before being able to display the recipes exactly as the demo offers:
https://demo.mealie.io/login

Thanks

**Comment by hay-kot** [comment, marked as answer]
Sure is, the nightly documentation has information on deploying the latest version where a login is now required. If you've already deployed the :latest tag, make sure you give the migration guide a viw

https://nightly.mealie.io/documentation/getting-started/introduction/

https://nightly.mealie.io/documentation/getting-started/migrating-to-mealie-v1/
